docker-compose.yml

```diff
@@ -45,9 +45,8 @@ services:
       # Users, products, product images and orgs files
       - users:/mnt/podata/users # .sto
       - products:/mnt/podata/products # .sto
+      - product_images:/mnt/podata/product_images # .jpg
```

Comment:
@ocervell I don't understand why you had to revert this ?
(the concern was the permissions, isn't it ?)